roles/stepup-selfservice/templates/parameters.yml.j2

```diff
@@ -106,8 +106,24 @@ parameters:
 {% endfor %}
 {% endfor %}
 
+{% if 'dev' in group_names %}
+    - slug: mock
+      name: 'Mock IDP'
+      logo: /images/remote-vetting/mock.png
+      description:
+          nl_NL: 'This is an integration test IdP.'
+          en_GB: 'This is an integration test IdP.'
+      entityId: 'https://selfservice.stepup.example.com/mock/metadata'
+      ssoUrl: 'https://selfservice.stepup.example.com/second-factor/mock/sso'
+      certificateFile: '%saml_rv_publickey%'
+      privateKey: '%saml_rv_privatekey%'
+      attributeMapping:
+          givenName: firstName
+          surname: lastName
+{% endif %}
+
     remote_vetting_sp:
-      entityId: "https://{{ selfservice_vhost_name }}/rv/metadata"
+      entityId: "https://{{ selfservice_vhost_name }}/saml/metadata"
```

Comment:
As discussed this configuration parameter should not exist: https://www.pivotaltracker.com/story/show/171928781